doc/getting_started/install.rst

```diff
@@ -25,91 +28,160 @@ The installed python is a full version of python. If your specialized
+This repo is the most useful for open scientific application development.
 
-Once your environment is in place, you can install directly from PyPI
-using pip::
+You can get an installer for your system (MacOS, Windows or Linux) here: https://github.com/conda-forge/miniforge?tab=readme-ov-file#miniforge3
 
-    pip install refl1d
+Run the installer, then from a terminal window (PowerShell or Terminal in Windows) run::
+
+    conda init 
+
+then close and re-open your terminal.
+
+
+Setup Environment
+-----------------
+
+You can create a new virtual environment for Refl1D, or install it in your base environment.  To create a new environment, run::
+
+    # if using miniforge
+    conda create -n refl1d
+    conda activate refl1d
 
-This will install refl1d, bumps and periodictable.
+    # if using regular python
+    python -m venv refl1d
+    source refl1d/bin/activate
 
-You can run the program by typing:
 
-    python -m refl1d.main
+Installation
+------------
 
-If this fails, then follow the instructions in :ref:`contributing` to install
-from the source archive directly.
+To install the application from source, clone the repository and install the
+dependencies::
 
-Windows
--------
+    git clone https://github.com/reflectometry/refl1d.git
+    cd refl1d
+    pip install .
 
-There are couple of options for setting up a python environment on windows:
+If you want to run the webview, you can install the optional extra::
 
-  - `python.org <https://www.python.org/>`_, and
-  - `Anaconda <https://www.anaconda.com/distribution/>`_.
+    pip install .[webview]
 
-With most pypi packages now bundled with wheels, it is now easy to set up a
-development environment using the official python package.  Similarly,
-anaconda provides binaries for all the refl1d dependencies.
+Additional Configuration - Windows 
+``````````````````````````````````
 
-You will need a C/C++ compiler.  If you already have Microsoft Visual C
+To install Refl1D on Windows, you will need a C/C++ compiler.  If you already have Microsoft Visual C
 installed you are done. If not, you can use the MinGW compiler that is supplied
 with your python environment or download your own.  You can set MinGW
 as the default compiler by creating the file *Lib\distutils\\distutils.cfg*
-in your python directory (e.g., *C:\\Python3.9*) with the following content::
+in your python directory (e.g., *C:\\Python3.10*) with the following content::
 
     [build]
     compiler=mingw32
 
-Once the python is prepared, you can install the periodic table and bumps
-package using the Windows console.
 
-Linux
------
+Additional Configuration - OS/X
+```````````````````````````````
 
-Linux distributions will provide the base required packages.  You
-will need to refer to your distribution documentation for details.
+Similar to windows, you can install the official python distribution or
+use Anaconda.  You will need to install the Xcode command line utilities
```

Comment:
as mentioned above, xcode is not needed.

Reply:
Miniforge is probably better, given that anaconda/miniconda require licences for most institutions.

Reply:
Agreed!  miniforge is great.

Reply:
https://discuss.scientific-python.org/t/response-to-anaconda-switch-to-paid-plans/1395/11